### ISSUE TITLE
Add support for downloading all failures to a CSV

### ISSUE THREAD

**Issue by hhowe29** [locustio/locust#675]
CSV should look about like the table in the Failures page.  Proposed columns

```
# Need these for sure
method
name
url
status_code
error_text

# these would be nice to have, but might be unwieldy to cram into a CSV
response_body  
request_headers
response_headers
```

**Comment by sanyco92** [comment]
Any updates on this issue?

**Comment by cgoldberg** [comment]
no PR for this has been submitted

**Comment by gboorse** [comment]
Hi @hhowe29 check the issue I made #864 

There is a workaround there that might help you.

**Comment by aldenpeterson-wf** [comment]
Addressed in #999
